cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -386,7 +394,8 @@ ReconciliationState createReconciliationState(Reconciliation reconciliation, Kaf
         /* test */ ReconcileResult<StatefulSet> zkDiffs;
         private Integer zkCurrentReplicas = null;
 
-        private KafkaCluster kafkaCluster = null;
+        KafkaCluster kafkaCluster = null;
```

Comment:
```suggestion
        /* test */ KafkaCluster kafkaCluster = null;
```
If you're omitting private, do you need it public for testing?